src/components/govukComponents/Button.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import ReactHtmlParser from 'react-html-parser'
+
+class Button extends Component {
+
+  render() {
+
+    var element = '';
+
+    if (this.props.element) {
+      element = this.props.element
+    } else if (this.props.href) {
+      element = 'a'
+    } else {
+      element = 'button'
+    }
+
+    var commonAttributes = { className: 'govuk-button' + (this.props.classes ? ' ' + this.props.classes : '') + (this.props.disabled ? ' govuk-button--disabled' : ''), ...this.props.attributes }
+
+    var buttonAttributes = { name: this.props.name, type: this.props.type }
+
+    if (element === 'a') {
+      var button = <a href={this.props.href} role="button" draggable="false" {...commonAttributes}>
```

Comment:
I can see this being a problem if we wanted the button to navigate to a react-router <Link> element. Maybe need to extend this slightly